deploy/helm/charts/README.md

```diff
@@ -96,59 +66,44 @@ helm install openebs-localpv openebs-localpv/localpv-provisioner --namespace ope
-| `openebsNDM.enabled`                        | Install openebs NDM dependency                    | `true`                        |
-| `rbac.create`                               | Enable RBAC Resources                             | `true`                        |
-| `rbac.pspEnabled`                           | Create pod security policy resources              | `false`                       |
+| Parameter                                   | Description                                                                                                                                                                                 | Default                       |
+| ------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| `release.version`                           | LocalPV Provisioner release version                                                                                                                                                         | `4.0.0`                       |
+| `analytics.enabled`                         | Enable sending stats to Google Analytics                                                                                                                                                    | `true`                        |
+| `analytics.pingInterval`                    | Duration(hours) between sending ping stat                                                                                                                                                   | `24h`                         |
+| `helperPod.image.registry`                  | Registry for helper image                                                                                                                                                                   | `""`                          |
+| `helperPod.image.repository`                | Image for helper pod                                                                                                                                                                        | `"openebs/linux-utils"`       |
+| `helperPod.image.pullPolicy`                | Pull policy for helper pod                                                                                                                                                                  | `"IfNotPresent"`              |
+| `helperPod.image.tag`                       | Image tag for helper image                                                                                                                                                                  | `3.5.0`                       |
```

Comment:
Let's bump this to 4.0.0 and make the corresponding change in the values as well? Linux-utils follows the localpv versioning scheme.

Reply:
We would need to release that first maybe then?

Reply:
Done.